scripts/verification/verify-dapi-management-data.js

```diff
@@ -0,0 +1,75 @@
+const assert = require('node:assert');
+const path = require('path');
+const { existsSync } = require('fs');
+const { ethers } = require('ethers');
+const forEach = require('lodash/forEach');
+const dapis = require('../../data/dapis.json');
+const currentHashData = require('../../data/dapi-management-merkle-tree-root/current-hash.json');
+const signerData = require('../../data/dapi-management-merkle-tree-root/hash-signers.json');
+const {
+  createDapiManagementMerkleTree,
+  deriveTreeHash,
+  getDapiManagementHashType,
+  deriveDataFeedId,
+  deriveSponsorWalletAddress,
+} = require('../utils');
+const { logSuccessMessage } = require('./utils');
+
+const SIGNATURE_KEYS_CHECK_ENABLED = false;
+
+function verifyData() {
+  console.info('Verifying dAPI Management data locally...');
```

Comment:
It is not really helpful to see `locally` in the CI. Could add confusion.
```suggestion
  console.info('Verifying dAPI Management data...');
```

Reply:
I understand, perhaps `Verifying dAPI Management local data...` but happy if you keep it as is 👍 

Thanks for the explanation